nltk/translate/bleu_score.py

```diff
@@ -106,16 +106,17 @@ def sentence_bleu(
     )
 
 
-def corpus_bleu(
-    list_of_references,
-    hypotheses,
-    weights=(0.25, 0.25, 0.25, 0.25),
-    smoothing_function=None,
-    auto_reweigh=False,
+def corpus_bleu_multiple_weights(
+        list_of_references,
+        hypotheses,
+        weights_list,
```

Comment:
There's no need to rename the class here. Also why is the variable `weights_list` more desired than the defaults `weights` that most users of BLEU expect?

Reply:
Ah the normal corpus_bleu has been moved down to another function.